docs/sphinx/source/user_guide/singlediode.rst

```diff
@@ -111,4 +111,4 @@ Clifford W. Hansen, Sandia `Report SAND2015-2065
 
 [4] "Computer simulation of the effects of electrical mismatches in
 photovoltaic cell interconnection circuits" JW Bishop, Solar Cell (1988)
-:doi:`10.1016/0379-6787(88)90059-2`
\ No newline at end of file
+:doi:`10.1016/0379-6787(88)90059-2`
```

Comment:
What changed here? It looks like nothing happened, but it shows as a change

Reply:
This seems to be an oversight on my part, I did not mean to alter that line.

Reply:
> We could link the references properly in line 20: by Jain and Kapoor _[1] _[2], and Hansen _[3]. 
> We could also standardize the formatting of the references

Only if @FelixKoTU wants to, in my view.

This page really doesn't fit in a User Guide, it's more of detailed background we ought to have somewhere but didn't know where else to put it. My point is that we ought to consider rewriting this page and editing the references can be done in that revision.

Reply:
> Only if @FelixKoTU wants to, in my view.
> 
> This page really doesn't fit in a User Guide, it's more of detailed background we ought to have somewhere but didn't know where else to put it. My point is that we ought to consider rewriting this page and editing the references can be done in that revision.

Fair point!